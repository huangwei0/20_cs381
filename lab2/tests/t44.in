debug off
stopwords off
length 10
load tracks.txt
preprocess
sequence amor
sequence corazon
sequence gusto
sequence grande
sequence triste