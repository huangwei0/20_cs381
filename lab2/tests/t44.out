amor de la vida es el sol y yo no more
corazon de la vida es el amor que te quiero ser
gusto es el amor de la vida loca pasion celeste aida
grande amor de la vida es el sol y yo no
triste navidad negra tomasa 
